support \r\n new lines

Raz0r opened the pull request:
This will help to correctly parse files with \r\n as new lines.

hynek commented:
Wouldn’t `\r?` be more idiomatic?  The change will also require a test that it really works with `\r\n`.

I’ve fixed the Travis failure, so could you also rebase on master please?

Thanks!

Raz0r commented:
Changed to ```\r?```, rebased on master, but Travis keeps failing (Twisted package checksum under pypy does not match, that's weird).
I will bother to write a test if I have time.

hynek commented:
fixed in d175ebf7b31eaf9dc917bb10e0d8cde2dcb5df34
